### PR TITLE
refactor: render audits outside of renderAuditGroup

### PR DESCRIPTION
**Pull request by patrickhulce**
R: @brendankenny 

cf. https://github.com/GoogleChrome/lighthouse/pull/2140#discussion_r115093860

### CODE REVIEW

File: lighthouse-core/report/v2/renderer/category-renderer.js [hunk 1]
```diff
@@ -162,16 +162,10 @@ class CategoryRenderer {
...
    * @param {!ReportRenderer.GroupJSON} group
-   * @param {function(!ReportRenderer.AuditJSON): !Element=} renderAudit
    * @return {!Element}
    */
-  _renderAuditGroup(audits, group, renderAudit) {
-    if (!renderAudit) {
-      renderAudit = this._renderAudit.bind(this);
-    }
-
+  _renderAuditGroup(group) {
```
Comment (brendankenny) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: maybe add a jsdoc description saying something about audits to be put in the group can be appended to the returned element?

Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/category-renderer.js]: done!

**Comment by patrickhulce** [comment]
ha did you edit the description to say 'cf. #issue', what's there to compare?

**Comment by brendankenny** [comment]
> ha did you edit the description 

ha, yeah, just to lead back to the "issue" that (sort of) explains the thinking behind the change

**Pull request merged**